src/main/java/socialite/logic/commands/EditCommand.java

```diff
@@ -46,8 +52,13 @@
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
             + "[" + PREFIX_ADDRESS + "ADDRESS] "
-            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "[" + PREFIX_REMARK + "REMARK] "
+            + "[" + PREFIX_TAG + "TAG]... "
+            + "[" + PREFIX_FACEBOOK + "FACEBOOK] "
+            + "[" + PREFIX_INSTAGRAM + "INSTAGRAM] "
+            + "[" + PREFIX_TELEGRAM + "TELEGRAM]\n"
```

Comment:
Maybe the `\n` should be shifted after `tiktok]`? And also the trailing whitespaces are missing here.